doc/contributing/distribution.md

```diff
@@ -25,3 +25,9 @@ only as a semver-major change, unless the related feature or project is
 documented as experimental. In addition, Node.js includes external projects as
 internal dependencies. These may be replaced or removed at any time, provided
 that doing so is not a breaking change.
+
+## Executables
+
+The Node.js distribution includes a number of executables, such as `node`. The
+distribution will only include executables whose names refer to software that is
```

Comment:
This is still incorrect if it tries to describe the status quo. `lib/node_modules/corepack/dist` contains other executables. There are probably a lot more in other nested `node_modules`. Users only really need to care about what's in `bin`. It's a common practice for runtimes to include additional executables that get invoked indirectly for certain use cases.

Reply:
What would be a succinct way of describing the status quo, then? How about this?

> The Node.js distribution includes a number of executables, such as `node`, that are intended to be run directly by users.

Reply:
I guess `bin` is what's supposed to be added to `PATH`, not the individual executable files.

Reply:
Do all platforms have `bin`? What would a wording be that includes all platforms?

But more broadly, why do we need to get into this level of detail? Is “The Node.js distribution includes a number of executables, such as node, that are intended to be run directly by users” not specific enough?

Reply:
Those executables can also be executed by "not-the-user", so we could remove that part. That leaves us with “The Node.js distribution includes a number of executables, such as node, that are intended to be run.” and that's not very informative, I don't think anyone would not expect to find executable in our distribution.

Reply:
I’ve updated it per https://github.com/nodejs/node/pull/51994#discussion_r1516642371, please let me know if you think this is inaccurate.